### PR TITLE
HTMD tutorial

### PR DESCRIPTION
**Pull request by chrisbarnettster**
@simonbray @tsenapathi @bgruening 
The layout of the tutorial created from the MD run and analysis workflow.  Still much more to do.

### CODE REVIEW

**Comment by simonbray** [comment]
Just made a few minor changes to the analysis section. Looks good already :+1:

**Comment by chrisbarnettster** [comment]
ping @simonbray, at last, it is out for review

**Comment by simonbray** [comment]
@chrisbarnettster, looks great. I don't have any other things to change.

Just one idea: is it worth keeping the mmgbsa section in the online version of the tutorial (maybe in an optional section, or in a 'hidden' box)? I think we should definitely do mmgbsa in the GCC training, even if we need to provide users with longer pre-generated trajectories for it.

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 1]
```diff
@@ -14,8 +14,9 @@ objectives:
 - Understand how different molecular interactions contribute to the binding affinity of various ligands for the Hsp90 protein.
 time_estimation: 3H
 key_points:
-- Simulating protein-ligand systems is more complex than simply simulating protein-only systems
-- ....
+- Simulating protein-ligand systems is more complex than simply simulating protein-only systems.
+- There are a range of Galaxy tools for MD simulation (using GROMACS) and analysis.
+- Galaxy makes assembling and scaling up workflows for high-throughput MD straightforward for users.
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:19]: I saw the key points were incomplete, so I wrote a few here:

**Comment by simonbray** [comment]
Okay, finally merged! Maybe we can discuss updating with the MMGBSA tool later.

**Pull request merged**